fastly/resource_fastly_service_v1.go

```diff
@@ -680,6 +680,16 @@ func resourceServiceV1() *schema.Resource {
 							Description:  "Where in the generated VCL the logging call should be placed.",
 							ValidateFunc: validateLoggingPlacement(),
 						},
+						"server_side_encryption": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Description:  "Specify where type of server side encryption should be used.",
+							ValidateFunc: validateLoggingServerSideEncryption(),
+						},
+						"server_side_encryption_kms_key_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
```

Comment:
Please add a `Description` field to this. This is the text we use in our [API docs](https://docs.fastly.com/api/logging#api-section-logging_s3):

> Optional server-side KMS Key Id. Must be set if `server_side_encryption` is set to `aws:kms`.